superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -132,12 +134,23 @@ const CoverFooterRight = styled.div`
   text-overflow: ellipsis;
 `;
 
+const ThinSkeleton = styled(Skeleton)`
+  h3 {
+    margin: 5 0;
```

Comment:
5px? Should probably also be based on gridUnit... which would normally be 4... so this seems a bit odd.